umm-spec-lib/src/cmr/umm_spec/test/iso_smap_expected_conversion.clj

```diff
@@ -73,8 +74,205 @@
+                  fix-archiver-distributor
+                  distinct)]
+    (assoc data-center :Roles roles)))
+
+(defn- order-data-centers
+  ""
+  [data-centers]
+  (let [grouped-data-centers (group-by #(= ["PROCESSOR"] (:Roles %)) data-centers)]
+    (concat (get grouped-data-centers false) (get grouped-data-centers true))))
+
+(defn- update-short-and-long-name
+  "ISO only has 1 field for both short and long and they get combined with a delimeter. combined
+  and then parse the short and long name here to mimic what UMM -> ISO -> UMM will do."
+  [data-center]
+  (let [{:keys [ShortName LongName]} data-center
+        organization-name (if LongName
+                           (str ShortName " &gt; " LongName)
+                           ShortName)
+        name-split (str/split organization-name #"&gt;|>")]
+    (if (> (count name-split) 0)
+     (-> data-center
+         (assoc :ShortName (str/trim (first name-split)))
+         (assoc :LongName (when (> (count name-split) 1)
+                           (str/join " " (map str/trim (rest name-split))))))
+     (-> data-center
+         (assoc :ShortName su/not-provided)
+         (assoc :LongName nil)))))
+
+(defn- update-person-names
+  "ISO only has one field for the whole name. When we go from UMM -> ISO, we combine the names into
+  one field then on ISO -> UMM we split them up. Need to do this processing to handle spaces in names
+  as well as leading/trailing spaces."
+  [person]
+  (let [{:keys [FirstName MiddleName LastName]} person
+        combined-name (str/trim (str/join " " [FirstName MiddleName LastName]))
+        names (str/split combined-name #" {1,}")
+        num-names (count names)]
+   (if (= 1 num-names)
+     (-> person
+         (assoc :LastName (first names))
+         (dissoc :FirstName)
+         (dissoc :MiddleName))
+     (-> person
+         (assoc :FirstName (first names))
+         (assoc :MiddleName (str/join " " (subvec names 1 (dec num-names))))
+         (update :MiddleName #(when (seq %) %)) ; nil if empty
+         (assoc :LastName (last names))))))
+
+(defn- expected-iso-contact-mechanisms
+ "Returns expected contact mechanisms with not translated types removed and ordered by phone,
+ fax, email"
+ [contact-mechanisms]
+ (when-let [contact-mechanisms (seq
+                                (remove #(nil? (:Type %))
+                                 (map #(assoc % :Type (get data-contact/translated-contact-mechanism-types (:Type %)))
+                                      contact-mechanisms)))]
+  (let [groups (group-by :Type contact-mechanisms)] ; Group for ordering
+    (concat
+     (get groups "Telephone")
+     (get groups "Fax")
+     (get groups "Email")))))
+
+(defn- expected-contact-info-related-urls
+  "Returns expected related url - take the first related url and the first url in related urls"
+  [related-urls]
+  (when related-urls
+    (cmr.umm-spec.test.iso19115-expected-conversion/expected-iso-19115-2-related-urls (take 1 related-urls))))
```

Comment:
It might be cleaner to do an alias for `iso19115-expected-conversion`. Generally, if the function is used by both it should go in conversion utils but I think this is fine.